spec/models/user_spec.rb

```diff
@@ -0,0 +1,117 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  subject {FactoryBot.create :user}
+
+  it "is valid factory" do
+    expect(subject).to be_valid
+  end
+
+  describe "associations" do
```

Comment:
phân biệt describe, context và it giúp anh với

Reply:
- describe: tạo ra một example group, bên trong có thể có nested group, sử dụng describe hoặc context
- context: là một bí danh của describe, context mô tả cho describe bọc ngoài nó, giúp cho test dễ đọc hơn
- it: Khai báo một example
----------
describe và context sử dụng trong trường hợp
describe: muốn tạo example group, muốn test các method, action, module, class
context: mô tả cho describe, dùng đánh giá các điều kiện khác nhau trong example group